pennylane/measurements.py

```diff
@@ -1221,6 +1251,12 @@ def circuit():
     return ShadowMeasurementProcess(Shadow, wires=wires, seed=seed)
 
 
+def shadow_expval(obs, k=1, seed_recipes=True):
+    """TODO: docs"""
```

Comment:
You can probably copy over most of the existing `device.shadow_expval` docstring :)

Reply:
yup missed that!